.github/workflows/build_and_deploy.yml

```diff
@@ -10,17 +10,36 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: golang:1.12
     steps:
-      - uses: actions/checkout@master
-      - name: dependencies
-        run: go mod download
-      - name: make test
-        run: make test
-      - name: release
+      -
+        name: checkout
+        uses: actions/checkout@master
+      -
+        name: setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13'
+      -
+        name: dependencies
+        run: |
+          go mod download
+      -
+        name: docker login
         env:
-          GITHUB_TOKEN: ${{ secrets.goreleaser }}
-        if: success()
+          DOCKER_USER: ${{ secrets.docker_user }}
+          DOCKER_PASSWORD: ${{ secrets.docker_password }}
+        run: |
+          echo $DOCKER_PASSWORD | docker login -u $DOCKER_USER --password-stdin
```

Comment:
Where are these credentials stored? Just curious ...

Reply:
In GitHub `Secrets`. Admins have access to remove or add new secrets.

Reply:
Huh. Intriguing. Things I still have to learn. :grin: